packages/eslint-plugin/tests/rules/unbound-method.test.ts

```
@@ -384,5 +410,38 @@ const unbound = new Foo().unbound;
         },
       ],
     },
+    {
+      code: `
+class BaseClass {
+  x: number = 42;
+  logThis() {
+    console.log('x is ');
+  }
+}
+
+class OtherClass extends BaseClass {
+  superLogThis: any;
+  constructor() {
+    super();
+    this.superLogThis = super.logThis;
+  }
+}
+
+const oc = new OtherClass();
+oc.superLogThis();
+      `,
+      options: [
+        {
+          allowSuper: false,
+        },
+      ],
+      errors: [
+        {
+          line: 13,
+          column: 25,
+          messageId: 'unbound',
+        },
+      ],
+    },
   ],
```

can you please add a sanity check, this should report an error

```ts
class BaseClass {
  x: number = 42;
  logThis() {
    console.log('x is ');
  }
}
class OtherClass extends BaseClass {
  superLogThis: any;
  constructor() {
    super();
    const x = super.logThis; // ERROR - unbound method
  }
}
```